app/api/schema/settings.py

```diff
@@ -196,3 +196,16 @@ class Meta:
     smtp_password = fields.Str(allow_none=True)
     smtp_port = fields.Integer(allow_none=True)
     smtp_encryption = fields.Str(allow_none=True)  # Can be tls, ssl, none
+
+    # Admin Invoice Details
+    admin_contact_name = fields.Str(allow_none=True)
+    admin_phone = fields.Str(allow_none=True)
+    admin_email = fields.Str(allow_none=True)
+    admin_country = fields.Str(allow_none=True)
+    admin_tax_info = fields.Str(allow_none=True)
+    admin_company = fields.Str(allow_none=True)
```

Comment:
instead of _company, let's follow the billing naming convention, it's more accurate.

Reply:
In this context, do you mean `admin_billing_company` ?

Reply:
um, rest of the fields, not this one. so let is be admin_compnay,

make others admin_billing_xxxx like we are doing for the other PR.

Reply:
Outdated